src/plugins/controls/common/control_group/types.ts

```diff
@@ -17,11 +17,14 @@ export interface ControlPanelState<TEmbeddableInput extends ControlInput = Contr
   width: ControlWidth;
 }
 
+export type ControlGroupChainingSystem = 'HIERARCHICAL' | 'NONE';
```

Comment:
That's a good question! I chose all caps for these values because it was originally an enum, and I changed it to a union string type because they're better DX-wise.

I am not sure if we have a naming convention for union string types, but I have no strong feelings one way or another, and could quite easily change it! Let's figure out which casing is used more frequently and update accordingly.